ArgusDocker/persistent/conf/argus-web-services/tomcat-users.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version='1.0' encoding='utf-8'?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<tomcat-users>
+	<!--
+	  NOTE:  By default, no user is included in the "manager-gui" role required
+	  to operate the "/manager/html" web application.  If you wish to use this app,
+	  you must define such a user - the username and password are arbitrary. It is
+	  strongly recommended that you do NOT use one of the users in the commented out
+	  section below since they are intended for use with the examples web
+	  application.
+	-->
+	<!--
+	  NOTE:  The sample user and role entries below are intended for use with the
+	  examples web application. They are wrapped in a comment and thus are ignored
+	  when reading this file. If you wish to configure these users for use with the
+	  examples web application, do not forget to remove the <!.. ..> that surrounds
+	  them. You will also need to set the passwords to something appropriate.
+	-->
+	<!--
+	  <role rolename="tomcat"/>
+	  <role rolename="role1"/>
+	  <user username="tomcat" password="<must-be-changed>" roles="tomcat"/>
+	  <user username="both" password="<must-be-changed>" roles="tomcat,role1"/>
+	  <user username="role1" password="<must-be-changed>" roles="role1"/>
+	-->
+	<role rolename="manager-gui"/>
+	<user username="argus" password="g3h31m" roles="manager-gui"/>
```

Comment:
I added these tomcat changes to the public docker PR and merged them in to kiwigrid-develop